app/gateway/changelog.md

```diff
@@ -9,6 +9,446 @@ Changelog for supported Kong Gateway versions.
 
 For product versions that have reached the end of sunset support, see the [changelog archives](https://legacy-gateway--kongdocs.netlify.app/enterprise/changelog/).
 
+## 3.6.0.0
+**Release Date** 02/12/2024
+
+### Breaking changes and deprecations
+
+* To avoid ambiguity with other Wasm-related `nginx.conf` directives, the prefix for Wasm `shm_kv` nginx.conf directives was changed from `nginx_wasm_shm_` to `nginx_wasm_shm_kv_`
+ [#11919](https://github.com/Kong/kong/issues/11919)
+
+* The listing endpoints for consumer groups (`/consumer_groups`) and consumers (`/consumers`) now respond
+  with paginated results. The JSON key for the list has been changed to `data` instead of `consumer_groups`
+  or `consumers`.
+
+* In OpenSSL 3.2, the default SSL/TLS security level has been changed from 1 to 2.
+  This means the security level is set to 112 bits of security. 
+  As a result, the following are prohibited:
+    * RSA, DSA, and DH keys shorter than 2048 bits
+    * ECC keys shorter than 224 bits
+    * Any cipher suite using RC4
+    * SSL version 3
+  Additionally, compression is disabled.
+
+* The recent OpenResty bump includes TLS 1.3 and deprecates TLS 1.1. 
+If you still need to still support TLS 1.1, set the [`ssl_cipher_suite`](/gateway/latest/reference/configuration/#ssl_cipher_suite) setting to `old`.
+
+* If you are using `ngx.var.http_*` in custom code in order to access HTTP headers, the behavior of that variable changed slightly when the same header is used multiple times in a single request. Previously it would return the first value only, now it returns all the values, separated by commas. Kong's PDK header getters and setters work as before.
```

Comment:
```suggestion
* If you are using `ngx.var.http_*` in custom code to access HTTP headers, the behavior of that variable changes slightly when the same header is used multiple times in a single request. 
Previously, it would return the first value only; now it returns all of the values, separated by commas. Kong Gateway's PDK header getters and setters work as before.
```